src/main/scala/slamdata/Publish.scala

```diff
@@ -16,7 +17,11 @@ class Publish {
       if (isSnapshot.value)
         Some("snapshots" at nexus + "content/repositories/snapshots")
       else
-        Some("releases"  at nexus + "service/local/staging/deploy/maven2")
+        sonatypeStagingRepositoryProfile.?.value map { stagingRepoProfile =>
+          "releases" at nexus +
+          "service/local/staging/deployByRepositoryId/" +
+          stagingRepoProfile.repositoryId
```

Comment:
Note for posterity: this means that `publishSigned` will *only* work when preceded by `sonatypeOpen`.  That's what we want for this plugin, so I don't think it's a problem, but it's something to be aware of.  A more general solution can be found here: https://github.com/xerial/sbt-sonatype/issues/55